### PR TITLE
Rearrange sidebar items

### PR DESCRIPTION
**Pull request by foxymiles**
**User-Facing Changes**
None (feature flagged)

**Description**
Rearranges the side bar items so that Settings & Topics are on the left and Events, Variables & Log Settings are on the right.

### CODE REVIEW

**Comment by jtbandes** [comment]
User-facing changes should be none.

**Review by 2metres — APPROVED** [review]
I think I prefer the Topics as the first item and the Panel Settings as the second. @amacneil thoughts?

File: packages/studio-base/src/Workspace.tsx [hunk 1]
```diff
@@ -599,24 +599,24 @@ function WorkspaceContent(props: WorkspaceProps): JSX.Element {
 
   const leftSidebarItems = useMemo(() => {
     const items = new Map<LeftSidebarItemKey, NewSidebarItem>([
+      ["panel-settings", { title: "Panel settings", component: PanelSettingsSidebar }],
```
Comment (amacneil) [inline packages/studio-base/src/Workspace.tsx:602]:
```suggestion
      ["panel-settings", { title: "Panel", component: PanelSettingsSidebar }],
```

@2metres wdyt? Seems cleaner imo, and still clear what it does

**Comment by amacneil** [comment]
No panel settings should come first, you need to interact with it constantly. The topics list is (currently) useless.

**Comment by amacneil** [comment]
Can you make the left sidebar open by default when the app is first launched.

**Pull request merged**